src/content/en/fundamentals/getting-started/service-worker/https-is-needed.markdown

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I might consider merging this into the Service worker prereqs doc, and add the appropriate headings. Tell people everything they require in a single doc, then have them move on.